src/cc65/codeopt.h

```diff
@@ -41,7 +41,7 @@
 /* cc65 */
 #include "codeseg.h"
 
-
+#define ANY "any"
```

Comment:
Please remove this macro again and just use - yet another identical - string constant for the printf().